src/services/refactors/moveToFile.ts

```diff
@@ -900,7 +900,7 @@ export function getUsageInfo(oldFile: SourceFile, toMove: readonly Statement[],
                         tryCast(decl, (d): d is codefix.ImportOrRequireAliasDeclaration => isImportSpecifier(d) || isImportClause(d) || isNamespaceImport(d) || isImportEqualsDeclaration(d) || isBindingElement(d) || isVariableDeclaration(d)),
                     ]);
                 }
-                else if (isTopLevelDeclaration(decl) && sourceFileOfTopLevelDeclaration(decl) === oldFile && !movedSymbols.has(symbol)) {
+                else if (!isGlobalType(checker, symbol) && isTopLevelDeclaration(decl) && sourceFileOfTopLevelDeclaration(decl) === oldFile && !movedSymbols.has(symbol)) {

```

Comment:
I'm not sure if this is perfect but, at the very least, it's no worse than the previous version. The preceding branch already handles `isInImport(decl)` and *those* should always be moveable. So I just pushed the existing check further down the road so the preceding branch can kick in in the case that is being fixed here.

Reply:
@andrewbranch What do you think of this vs #60173, i think it is the same bug